lib/screens/mycapital_screens/my_assets_screen.dart

```diff
@@ -0,0 +1,161 @@
+import 'dart:convert';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:fl_chart/fl_chart.dart';
+import 'package:intl/intl.dart';
+import 'package:jusicool_design_system/src/core/theme/texts/typography.dart';
+import 'package:jusicool_design_system/src/core/theme/colors/color_palette.dart';
+import 'package:jusicool_ios/models/my_assets.dart';
+import 'package:jusicool_ios/widgets/my_asset_tile.dart';
+
+class MyAssetsScreen extends StatefulWidget {
+  const MyAssetsScreen({super.key});
+
+  @override
+  State<MyAssetsScreen> createState() => _MyAssetsScreenState();
+}
+
+class _MyAssetsScreenState extends State<MyAssetsScreen> {
+  late Future<MyAssetsData> _futureData;
+
+  @override
+  void initState() {
+    super.initState();
+    _futureData = _loadAssetsData();
+  }
+
+  Future<MyAssetsData> _loadAssetsData() async {
+    final jsonString = await rootBundle.loadString(
+      'assets/data/my_assets.json',
+    );
+    final Map<String, dynamic> jsonMap = json.decode(jsonString);
+    return MyAssetsData.fromJson(jsonMap);
+  }
```

Comment:
실제 출시할때 사용할 코드가 아니라면 

```suggestion
/// ==================================== 
  Future<MyAssetsData> _loadAssetsData() async {
    final jsonString = await rootBundle.loadString(
      'assets/data/my_assets.json',
    );
    final Map<String, dynamic> jsonMap = json.decode(jsonString);
    return MyAssetsData.fromJson(jsonMap);
  }
  /// ==================================== 
```

이런식으로 해도 될거같아요

Reply:
다른 코드들도 실제 사용할 코드가 아니라면 저런식으로 해도 될거같아요

Reply:
반영했습니다